prov/sm2/src/sm2.h

```diff
@@ -208,36 +207,36 @@ struct sm2_queue {
 };
 
 struct sm2_fabric {
-	struct util_fabric	util_fabric;
+	struct util_fabric util_fabric;
 };
 
 struct sm2_domain {
-	struct util_domain	util_domain;
-	int			fast_rma;
+	struct util_domain util_domain;
+	int fast_rma;
 	/* cache for use with hmem ipc */
-	struct ofi_mr_cache	*ipc_cache;
-	struct fid_peer_srx	*srx;
+	struct ofi_mr_cache *ipc_cache;
+	struct fid_peer_srx *srx;
 };
 
-#define SM2_PREFIX	"fi_shm://"
-#define SM2_PREFIX_NS	"fi_ns://"
+#define SM2_PREFIX "fi_shm://"
+#define SM2_PREFIX_NS "fi_ns://"
 
-#define SM2_ZE_SOCK_PATH	"/dev/shm/ze_"
+#define SM2_ZE_SOCK_PATH "/dev/shm/ze_"
 
-#define SM2_RMA_ORDER (OFI_ORDER_RAR_SET | OFI_ORDER_RAW_SET | FI_ORDER_RAS |	\
-		       OFI_ORDER_WAR_SET | OFI_ORDER_WAW_SET | FI_ORDER_WAS |	\
-		       FI_ORDER_SAR | FI_ORDER_SAW)
-#define sm2_fast_rma_enabled(mode, order) ((mode & FI_MR_VIRT_ADDR) && \
-			!(order & SM2_RMA_ORDER))
+#define SM2_RMA_ORDER                                                                    \
+(OFI_ORDER_RAR_SET | OFI_ORDER_RAW_SET | FI_ORDER_RAS | OFI_ORDER_WAR_SET |              \
+ OFI_ORDER_WAW_SET | FI_ORDER_WAS | FI_ORDER_SAR | FI_ORDER_SAW)
```

Comment:
Don't love how it does these. Would be interested in modifying but I'm not passionate about it.

Reply:
Looks like you did manage to fix these in clang-format file.